roles/utils_build_push_image/tasks/main.yml

```diff
@@ -0,0 +1,200 @@
+- name: Check that local_image_name is defined
+  fail: msg="Bailing out. This play requires 'local_image_name'"
+  when: local_image_name is undefined
+
+- name: Check that image_tag is defined
+  fail: msg="Bailing out. This play requires 'image_tag'"
+  when: image_tag is undefined
+
+- name: Check that branch is defined
+  fail: msg="Bailing out. This play requires 'branch'"
+  when: branch is undefined
+
+- name: Check that artifact_extra_logs_dir is defined
+  fail: msg="Bailing out. This play requires 'artifact_extra_logs_dir'"
+  when: artifact_extra_logs_dir is undefined
+
+- name: Check that memory is defined
+  fail: msg="Bailing out. This play requires 'memory'"
+  when: memory is undefined
```

Comment:
will this work with no `memory` is passed to the CLI?

and could you add a file `config/main.yml` like this [one](https://github.com/openshift-psap/ci-artifacts/blob/master/roles/gpu_operator_bundle_from_commit/defaults/main/config.yml) to list the parameters expected by the role?